docs/assets/css/fluid.css

```diff
@@ -240,6 +240,8 @@ where bootstrap caption color is too light for white background */
  Long words such as function names would not break in api docs causing layout overflow
 */
 
-.table td:nth-child(2) {
-	word-break: break-word;
+@media only screen and (max-width: 600px) {
+	.table td:nth-child(2) {
```

Comment:
Interesting that this is only for the second column. The tables generated for the API docs are somewhat dynamic. E.g., if there are no `Alerts` contents for any of the items in a table, that column will be omitted altogether, and a different column will be the second one.

It looks like the original use-case was targeting the API member names specifically, which will generally be the 1st column in API docs tables. We may want to target the 1st column instead of the 2nd. And we should expand on the above documentation to explain _why_ the first column is the appropriate one.